get_letters.py

```diff
@@ -0,0 +1,10 @@
+def get_letters() -> list[str]:
+    """
+    This function return all the letters lowercase and uppercase using list comprehension.
```

Comment:
“Code tells you how; Comments tell you why.”
https://realpython.com/documenting-python-code/